generators/frontend/templates/react-demo/components/Header.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import Link from 'next/link'
+import Util from '../common/Util'
+import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faStar, faHandSpock, faShippingFast, faHeart, faAngleDown } from '@fortawesome/fontawesome-free-solid'
+
+export default class extends Component {
+  render() {
+    const Wrapper = styled.div`
+      margin: 20px;
```

Comment:
When you write css it will be easier to read if you specify flex stuff together, then margins, paddings. So as example I would rewrite this css: 
```css
display: flex;
flex-direction: row;
justify-content: space-between;
width: contain;
margin: 20px;
list-style: none;
```
Imagine you read the story about the block, so first you want to read it is flex and other settings of flex, them block margins and then other stuff. If you read css about header or pharagraph, you probably would like to read first about font-family, font-size and then about margins etc.